libr/util/p_format.c

```diff
@@ -349,12 +349,11 @@ static void r_print_format_decchar(const RPrint* p, int endian, int mode,
 
 static int r_print_format_string(const RPrint* p, ut64 seeki, ut64 addr64, ut64 addr, int is64, int mode) {
 	ut8 buffer[255];
+	bool can_read;
```

Comment:
Initialize to false

Reply:
There is no path where `can_read` is not assigned.

Reply:
This file is already a hell, as @alvarofe said.